frontend/packages/component-library/src/dropdown/CustomDropdown.tsx

```diff
@@ -189,13 +203,15 @@ const CustomDropdown: React.FunctionComponent<CustomDropdownProps> = ({
           [moduleStyles.hasError]: errorMessage,
           [moduleStyles.readOnly]: readOnly,
           [moduleStyles.styleAsFormField]: styleAsFormField,
+          [moduleStyles.fullWidth]: fullWidth && !width, // Only apply if no custom width
```

Comment:
```suggestion
          [moduleStyles.fullWidth]: fullWidth,
```

as the rest here I think is redundant,
inline styles value of width will always be a higher priority selector anyways since it's inline styles

Reply:
Good point, removed the && !width guard here and in simple dropdown as well!